framework/lib/components/combo-picker/combo-picker.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useState } from 'react'
+import { Button, InputGroup, InputRightElement, NumberInputField } from '@chakra-ui/react'
+import { identity } from 'ramda'
+import { ComboPickerProps } from './types'
+import { Select } from '../select'
+import { NumberInput } from '../number-input'
+
+/**
+ * @see ComboPicker
+ * @see {@link https://northlight.dev/reference/combo-picker}
+ */
+export const ComboPicker = ({
+  onChange = identity,
+  options,
+  defaultOption,
+  size,
+  value = { input: 0, option: defaultOption || options[0] },
```

Comment:
pretty hard to read when defaulting props depending on previous prop like options[0] maybe cold default in the body of function or just do nullish coa `??` when its used